src/components/shared/ParamField.vue

```diff
@@ -1,22 +1,31 @@
 <script setup lang="ts">
 import { reactive, computed, watch } from 'vue'
-import { toWei, Unit, padLeft, numberToHex } from 'web3-utils'
-import ERC725 from '@erc725/erc725.js'
+import { toWei, type Unit, padLeft, numberToHex } from 'web3-utils'
+import ERC725, { type ERC725JSONSchema } from '@erc725/erc725.js'
 import LSPSelect from '@/components/shared/LSPSelect.vue'
 import { BN } from 'bn.js'
-import { MethodType } from '@/helpers/functionUtils'
-import { TokenInfo } from '@/helpers/tokenUtils'
+import type { MethodType } from '@/helpers/functionUtils'
+import type { TokenInfo } from '@/helpers/tokenUtils'
+import LSP3ProfileMetadata from '@erc725/erc725.js/schemas/LSP3ProfileMetadata.json'
+import LSP4DigitalAsset from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json'
+import LSP9Vault from '@erc725/erc725.js/schemas/LSP9Vault.json'
+
+const schemas = LSP3ProfileMetadata.concat(LSP4DigitalAsset, LSP9Vault)
```

Comment:
Would suggest writing it as follow for readability (easier to see the schemas in there when they are written line by line. But not a big deal.

```js
const schemas = [
    ...LSP3ProfileMetadata,
    ...LSP4DigitalAsset, 
    ...LSP9Vault
]
```

Reply:
Changing